python/altrios/__init__.py

```diff
@@ -104,15 +104,17 @@ class (relative to the class the method was called on)
     specified by user. Specifies a list of paths to be appended to the list
     returned by the method.
     """
-    variable_list = [attr for attr in self.__dir__() if not attr.startswith("__") and not callable(getattr(self,attr))]
+    variable_list = [attr for attr in self.__dir__() if not attr.startswith("__")\
+         and not callable(getattr(self,attr))]
     for variable in variable_list:
         if not type(getattr(self,variable)).__name__ in ACCEPTED_RUST_STRUCTS:
             if path == "":
                 variable_path = variable
             else:
                 variable_path = path + "." + variable
             param_path_list.append(variable_path)
-        elif len([attr for attr in getattr(self,variable).__dir__() if not attr.startswith("__") and not callable(getattr(getattr(self,variable),attr))]) == 0:
```

Comment:
@robinsteuteville for long lines, you can shorten them by ending with `\`.  Ensure that the `\` is right after the last character of the code on that line and that there is no trailing whitespace.